number/src/macros.rs

```diff
@@ -23,12 +23,14 @@ macro_rules! powdr_field {
             Hash,
             Serialize,
             Deserialize,
+            JsonSchema,
         )]
         pub struct $name {
             #[serde(
                 serialize_with = "crate::serialize::ark_se",
                 deserialize_with = "crate::serialize::ark_de"
             )]
+            #[schemars(skip)]
             value: $ark_type,
```

Comment:
I cannot find any way to include this field in the `JsonSchema`, I'm wondering if this would be safe to skip it altogether considering how deep it is tight to the types in `powdr`. It means that when upgrading to another major `ark` version that breaks backward compatibility, one would have to bump the `analyzed_type.version` file manually.
It might be an edge case that is not worth pursuing but I'm not really sure

Reply:
I think it's OK.

There are limited ways to store BigInt / FieldElement / Elliptic curves and the serialization they use should be stable due to the size of their ecosystem.

They just dump bigint in little-endian: https://github.com/arkworks-rs/algebra/blob/cc2ad8c368c323ee9d9aacaabf8e353de95682e6/serialize/src/impls.rs#L170-L181

Reply:
And coincidentally Halo2 also uses little-endian.

Reply:
Yea little-endian is rather common across the board